roles/iommu/README.md

```diff
@@ -4,6 +4,10 @@
 
 - `iommu_pt`: Default true, Configure passthrough mode, which doesn't require DMA translation.
 
+## Host Vars
+
+- `vfio_pci_ids`: Can optionally be set with the pci id of the device to pass-through.
```

Comment:
Would it be worth using the iommu prefix to stop variable collisions? Also I guess it can go in the "Variable" section of the README with the other variable.